packages/docs/getting-started/running-a-validator.md

```diff
@@ -116,11 +118,12 @@ First we are going to setup the main environment variables related with the `Bak
 
 ```bash
 export CELO_IMAGE=us.gcr.io/celo-testnet/celo-node:baklava
-export NETWORK_ID=12219
+export NETWORK_ID=76172
 ```
 
 ### Pull the Celo Docker image
 
+<a name="pull-image"></a>
```

Comment:
The name attribute on the <a> tag is not supported in HTML5. We should use a <span> tag with an id attribute instead.